pkg/asset/agent/common.go

```diff
@@ -0,0 +1,21 @@
+package agent
+
+import (
+	"github.com/openshift/installer/pkg/types/baremetal"
+	"github.com/openshift/installer/pkg/types/none"
+	"github.com/openshift/installer/pkg/types/vsphere"
+)
+
+// SupportedPlatforms lists the supported platforms for agent installer
+var SupportedPlatforms = []string{baremetal.Name, vsphere.Name, none.Name}
+
+// IsSupportedPlatform returns true if provided platform is baremeral, vsphere or none.
```

Comment:
Typo (baremeral), but I'd recommend just removing this comment as it just reiterates the code 2 lines above and makes it harder to maintain when we support other platforms

Reply:
linter will complain if you remove it altogether, but agree that there's no need to list the platforms here.